src/Discord/Parts/User/User.php

```diff
@@ -169,8 +171,10 @@ public function getBannerAttribute(string $format = 'jpg', int $size = 600): ?st
             return null;
         }
 
-        if (false === array_search($format, ['png', 'jpg', 'webp', 'gif'])) {
-            $format = 'jpg';
+        $allowed = ['png', 'jpg', 'webp', 'gif'];
+	
+		if (! in_array(strtolower($format), $allowed)) {
+            $format = 'webp';
```

Comment:
Use 8 spaces instead of 2 tabs
```suggestion
        if (! in_array(strtolower($format), $allowed)) {
            $format = 'png';
```
Banner default format is png

```php
public function getBannerAttribute(string $format = 'png', int $size = 600): ?string
```